__docs__/wonder-blocks-link/link.stories.tsx

```diff
@@ -393,8 +393,8 @@ Inline.parameters = {
 Inline.play = async ({canvasElement}) => {
     const canvas = within(canvasElement);
 
-    const primaryLink = canvas.getByText("Primary link");
-    const secondaryLink = canvas.getByText("Secondary link");
```

Comment:
These were capturing the `span` that was inside the `a` tag instead of the `a` tag itself.